pkg/cmd/join/cmd.go

```diff
@@ -62,5 +62,7 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 	cmd.Flags().BoolVar(&o.wait, "wait", false, "If true, running the cluster registration in foreground.")
 	cmd.Flags().StringVarP(&o.mode, "mode", "m", "default", "mode to deploy klusterlet, can be default or hosted")
 	cmd.Flags().StringVar(&o.managedKubeconfigFile, "managed-cluster-kubeconfig", "", "To specify the directory to external managed cluster kubeconfig in hosted mode")
+	cmd.Flags().BoolVar(&o.standalone, "standalone", false, "If true, deploy standalone controlplane agent instead of klusterlet")
```

Comment:
we should not call at as `standalone`. it is misleading. maybe we just use `multicluster-controlplane` instead.

Reply:
IMHO, it is hard to understand what is multicluster-controlplane. It is a project name rather than how we deploy the agent. And in the long term, we probably will build the standalone agent directly from the ocm repo.